doc/api/child_process.md

```diff
@@ -293,6 +293,12 @@ added: v0.1.90
 * `options` {Object}
   * `cwd` {String} Current working directory of the child process
   * `env` {Object} Environment key-value pairs
+  * `argv0` {String} Explicitly set the value of `argv[0]` sent to the child
+    process. This will be set to `command` if not specified. *Note:* Certain
+    platforms (OS X, Linux) will use the value of `argv[0]` for the process
+    name while others (Windows, SunOS) will use `command`. *Note:* Node
```

Comment:
I’d maybe say `… for the process title` so it’s clear that refers to `process.title`.